README.md

```diff
@@ -363,24 +363,6 @@ Translations of the guide are available in the following languages:
            else
              calc_something_else
            end
-
-  # good (and a bit more width efficient)
-  kind =
-    case year
-    when 1850..1889 then 'Blues'
-    when 1890..1909 then 'Ragtime'
-    when 1910..1929 then 'New Orleans Jazz'
-    when 1930..1939 then 'Swing'
-    when 1940..1950 then 'Bebop'
-    else 'Jazz'
-    end
```

Comment:
This style of spacing is used later on in a section we already went over https://github.com/goldstar/ruby-style-guide/pull/1/files?diff=unified#diff-04c6e90faac2675aa89e2176d2eec7d8R761